frontend-gd/src/main/tools/fetch.js

```diff
@@ -1,3 +1,9 @@
+// Took this from master because all the pages were getting this error:
```

Comment:
Pretty sure this whole file can be deleted.

Reply:
Done.